ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java

```diff
@@ -81,6 +73,14 @@ public void uploadFile(String extensionName) throws Throwable {
         $(By.cssSelector("input[type='file']")).shouldBe(visible).uploadFile(techExtensionJar.toFile());
     }
 
+    @When("^Camilla upload extension with name \"([^\"]*)\" from relative path \"([^\"]*)\"$")
+    public void uploadExtensionFromFile(String extensionName, String extensionPath) throws Throwable {
```

Comment:
Very similar to `uploadFile()` method, maybe an overload?

Reply:
`uploadFile()` should be deprecated and refactored as it is legacy and only works for one extension - syndesis-extension-log-body

Reply:
Lets to this clean cut then...

Reply:
I like to have a step that is able to upload with name, from `syndesis-extensions` directory. It's not a lot of changes though to get it working.